Platform/AlderlakeBoardPkg/Library/FspsUpdUpdateLib/FspsUpdUpdateLib.c

```diff
@@ -387,6 +387,10 @@ TccModePostMemConfig (
     FspsUpd->FspsConfig.TccCacheCfgBase = (UINT32)(UINTN)TccCacheconfigBase;
     FspsUpd->FspsConfig.TccCacheCfgSize = TccCacheconfigSize;
     DEBUG ((DEBUG_INFO, "Load Tcc Cache @0x%p, size = 0x%x\n", TccCacheconfigBase, TccCacheconfigSize));
+
+    if (IsPchS ()) {
```

Comment:
do we really need this check?

Reply:
I will remove it later once I can validate on other ADL flavors.

Reply:
Inside FSP it should have such check (so this kind of UPD would not be used if TCC is not supported) from my understanding. OK, let's merge it firstly.